junos/resource_interface_physical.go

```diff
@@ -516,6 +556,11 @@ func readInterfacePhysical(interFace string, m interface{}, jnprSess *NetconfObj
 
 			case strings.HasPrefix(itemTrim, "ether-options 802.3ad "):
 				confRead.v8023ad = strings.TrimPrefix(itemTrim, "ether-options 802.3ad ")
+			case strings.HasPrefix(itemTrim, "esi "):
+				confRead.Esi, err = readIntEsi(itemTrim, confRead.Esi)
+				if err != nil {
+					return confRead, err
+				}
```

Comment:
Insert condition to respect alphabetical order (after if description)
Use new suggest parameters 
```suggestion
			case strings.HasPrefix(itemTrim, "esi "):
				if err := readIntEsi(&confRead, itemTrim); err != nil {
					return err
				}
```